### PR TITLE
apt-get: add Indonesian translation

### PR DESCRIPTION
**Pull request by punkestu**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/linux/apt-get.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get install {{package}}`
+
+- Menghapus sebuah paket:
+
+`apt-get remove {{package}}`
+
+- Menghapus sebuah paket dan file konfigurasinya:
+
+`apt-get purge {{package}}`
```
Comment (kbdharun) [inline pages.id/linux/apt-get.md:21]:
```suggestion
`apt-get install {{paket}}`

- Menghapus sebuah paket:

`apt-get remove {{paket}}`

- Menghapus sebuah paket dan file konfigurasinya:

`apt-get purge {{paket}}`
```

**Ref.** https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution. I have a minor suggestion for this page.

We generally translate possible placeholders inside `{{}}` too.

**Review by kbdharun — APPROVED** [review]

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks contributing! While the content itself is OK, there are some few changes you'll need to comply with our [style guide](https://share.reinhart1010.id/tldrstyleguide).

File: pages.id/linux/apt-get.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+> Manajemen paket untuk Debian dan Ubuntu.
+> Cari paket menggunakan `apt-cache`.
+> Informasi lebih lanjut: <https://manpages.debian.org/latest/apt/apt-get.8.html>.
+
+- Perbarui daftar paket yang tersedia beserta versinya (hal ini direkomendasikan untuk dijalankan sebelum menjalankan perintah `apt-get` yang lain):
+
+`apt-get update`
+
+- Menginstal sebuah paket, atau memperbaruinya ke versi terbaru yang tersedia:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:11]:
```suggestion
- Pasang sebuah paket, atau perbarui ke versi terbaru yang tersedia:
```

File: pages.id/linux/apt-get.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get install {{paket}}`
+
+- Menghapus sebuah paket:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:15]:
```suggestion
- Hapus sebuah paket:
```

File: pages.id/linux/apt-get.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get remove {{paket}}`
+
+- Menghapus sebuah paket dan file konfigurasinya:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:19]:
```suggestion
- Hapus sebuah paket dan file konfigurasinya:
```

File: pages.id/linux/apt-get.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get purge {{paket}}`
+
+- Memperbarui semua paket yang terinstal ke versi terbaru yang tersedia:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:23]:
```suggestion
- Perbarui semua paket yang terpasang ke versi terbaru yang tersedia:
```

File: pages.id/linux/apt-get.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get upgrade`
+
+- Membersihkan repositori lokal - menghapus file paket (`.deb`) yang sebelumnya gagal didownload dan tidak bisa di download lagi:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:27]:
```suggestion
- Bersihkan repositori lokal, hapus file paket (`.deb`) yang sebelumnya gagal diunduh dan tidak bisa diunduh kembali:
```

File: pages.id/linux/apt-get.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get autoclean`
+
+- Menghapus semua paket yang tidak diperlukan lagi:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:31]:
```suggestion
- Hapus semua paket yang tidak diperlukan kembali:
```

File: pages.id/linux/apt-get.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get autoremove`
+
+- Memperbarui paket yang terinstal (mirip `upgrade`), tapi bedanya ini menghapus paket yang tidak dipakai lagi dan menginstal paket tambahan untuk memenuhi dependensi baru:
```
Comment (reinhart1010) [inline pages.id/linux/apt-get.md:35]:
```suggestion
- Perbarui paket yang terinstal (mirip `upgrade`), namun hapus paket yang tidak dipakai kembali dan pasang paket tambahan untuk memenuhi dependensi baru:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Hi, can you also apply these changes? Thank you!

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Pull request merged**